components/carousel/script.js

```diff
@@ -80,6 +87,22 @@ class Carousel extends BaseComponent {
         return this.querySelector(CONTENT_WRAPPER_SELECTOR).children;
     }
 
+    /**
+     * Set the array of items to the carousel
+     * @param {Array} value
+     */
+    set items(value) {
+        const contentWrapper = this.querySelector(CONTENT_WRAPPER_SELECTOR);
+        contentWrapper.innerHTML = '';
+
+        value.forEach(element => contentWrapper.appendChild(element));

```

Comment:
Do we need here to validate the items to prevent errors if the setter is called with an invalid value? For example, it is not supposed to do `carousel.items = 'invalid'`, `carousel.items = ['invalid']`

Reply:
Yes, we should - added.